### ISSUE TITLE
训练效果有偏差

### ISSUE THREAD

**Issue by wokeyide1999** [Xiaoqi-Zhao-DLUT/GateNet-RGB-Saliency#7]
我在数据集DUTS上进行10000次迭代的情况下，分别进行了无Fold-ASPP模块和有Fold-ASPP模块的训练；
进行无Fold-ASPP的修改为将self.dem1修改为如下代码
`self.dem1 = nn.Sequential(nn.Conv2d(2048, 512, kernel_size=3, padding=1), nn.BatchNorm2d(512), nn.PReLU())`
同时将self.layer4的stride调整为2
`self.layer4 = self._make_layer(block1, 512, layers[3], stride=2)`

我的预期效果是有Fold-ASPP的训练好于无Fold-ASPP的训练效果，但结果恰好相反，请问可能的原因是什么？(训练得到的测试结果如下)
迭代10000次，无fold-ASPP

DUTS-TE MAE: 0.0511 maxF: 0.8533 avgF: 0.7539 wfm: 0.7512 Sm: 0.8570 Em: 0.8677

迭代10000次，有Fold-ASPP

DUTS-TE MAE: 0.0526 maxF: 0.8468 avgF: 0.7622 wfm: 0.7505 Sm: 0.8523 Em: 0.8735

**Comment by Xiaoqi-Zhao-DLUT** [comment]
@wokeyide1999 迭代10000次? 起码要40epoch or 100000 次～

**Comment by wokeyide1999** [comment]
> @wokeyide1999 迭代10000次? 起码要40epoch or 100000 次～

明白了，多谢
